src/components/Authenticated/Authenticated.jsx

```diff
@@ -1,101 +1,105 @@
-import React, { Component } from 'react';
-import conf from '../../config/config.json';
+import React, { Component } from 'react'
+import conf from '../../config/config.json'
 
-const SESSION_URI = conf.services[0] + '/_session';
+const SESSION_URI = conf.services[0] + '/_session'
```

Comment:
Idem.

Reply:
I'm fed up. Please scan each diff line yourself. And check EVERY rule before requesting for a pull.